app/src/CoursePages/CoursePage.tsx

```diff
@@ -30,33 +30,96 @@ export const CoursePage: React.FC = () => {
         fetchData();
     }, [classNum]);
 
+    return (
+        <div className="coursepage">
+            <div className="topbox">
+                <CourseInfo classNum={ classNum! }
+                courseName={ course.length > 0 ? course[0].name : '...' }
+                desc={ course.length > 0 ? course[0].description : 'Loading...' }/>
+                <div className="rightflex">
+                    <h3 className="overallratings-header">Overall Ratings</h3>
+                    <div className="ratings-flexbox">
+                        <OverallRatingBox label="Difficulty" rating={ course.length > 0 ? course[0].rating_one : '?/5' }></OverallRatingBox>
+                        <OverallRatingBox label="Workload" rating={ course.length > 0 ? course[0].rating_two : '?/5' }></OverallRatingBox>
+                        <OverallRatingBox label="Practicality" rating={ course.length > 0 ? course[0].rating_three : '?/5' }></OverallRatingBox>
+                    </div>
+                </div>
+            </div>
+
+            <div className="bottombox">
+                <div className="page-reviews-header">User Reviews</div>
+                <ReviewHolder classNum={classNum!}/>
+            </div>
+        </div>
+    );
+}
+
+interface CourseInfoProps {
+    classNum: string;
+    courseName: string;
+    desc: string;
+}
+
+export const CourseInfo: React.FC<CourseInfoProps> = (props) => {
     const navigate = useNavigate();
 
-    const handleClick = () => {
-        navigate('/course/' + classNum + '/review')
+    const rateButtonClick = () => {
+      navigate('/course/' + props.classNum + '/review')
+    }
+
+    const courseButtonClick = () => {
+      const url = 'https://courses.cs.washington.edu/courses/cse' + props.classNum + '/';
+
+      window.open(url, '_blank');
+    }
+
+    const dawgButtonClick = () => {
+      const url = 'https://dawgpath.uw.edu/course?id=CSE+' + props.classNum + '&campus=seattle';
+
+      window.open(url, '_blank');
     }
 
     return (
-      <div className="coursepage">
-        <div className="topbox">
-          {course.map((courseObject) => (
-            <>
-            <p>{ courseObject.number }</p>
-            <p>{ courseObject.name }</p>
-            <p>{ courseObject.description }</p>
-            <p>{ courseObject.rating_one }</p>
-            <p>{ courseObject.rating_two }</p>
-            <p>{ courseObject.rating_three }</p>
-            </>
-          ))}
-          <button onClick={handleClick}>Review</button>
+        <div className="courseinfo">
```

Comment:
Can you camel case these? I think the style guide wants camel casing

Reply:
It's confusing because the Google style guide for the HTML/CSS says to use all lowercase for classnames, but in the TypeScript one it says to use camel case. In the rest of the code I think all the classNames are in lower case and words are separated by a hyphen. I changed my classnames to be more readable, but I kept them all lowercase. Let me know if you want me to change all the classnames to camel case in the other files too.

Reply:
It’s fine then. It doesn’t matter that much